Examples/SwiftExample/watchSample Extension/InterfaceController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  InterfaceController.swift
+//  watchPurchasesCocoapods2 WatchKit Extension
+//
+//  Created by RevenueCat on 2/11/20.
+//  Copyright © 2020 RevenueCat. All rights reserved.
+//
+
+import WatchKit
+import Foundation
+import Purchases
+
+class InterfaceController: WKInterfaceController {
+    private var offering : Purchases.Offering?
+    
+    @IBOutlet weak var expiryDateLabel: WKInterfaceLabel!
+    @IBOutlet weak var purchaseDateLabel: WKInterfaceLabel!
+    @IBOutlet weak var proStatusLabel: WKInterfaceLabel!
+    @IBOutlet weak var buyButton: WKInterfaceButton!
+    
+    override func awake(withContext context: Any?) {
+        super.awake(withContext: context)
+        configure()
+        loadOfferings()
+        // Configure interface objects here.
+    }
+    
+    override func willActivate() {
+        // This method is called when watch view controller is about to be visible to user
+        super.willActivate()
+    }
+    
+    override func didDeactivate() {
+        // This method is called when watch view controller is no longer visible
+        super.didDeactivate()
+    }
+    
+    
+    @IBAction func didPressBuy() {
+        purchase()
+    }
+}
+
+private extension InterfaceController {
+    
+    func purchase() {
+        guard let offering = offering else {
+            print("No available offerings")
+            return
+        }
+        let package = offering.availablePackages[0]
```

Comment:
Maybe I would get a specific package here instead. That way we know which one, in particular, will be purchased.